.github/workflows/cluster-setup/action.yaml

```diff
@@ -4,21 +4,24 @@ inputs:
   github-token:
     description: GITHUB_TOKEN from the calling workflow
     required: true
+  custom-ns:
```

Comment:
why a bool and not a string so that we can choose the name of the namespace?

Reply:
actually, it is a string only, ` custom-ns: "true"` this is just passing a var if we want to deploy on custom namespace or not.

Reply:
okay, We have 2 namespaces so I'll create another input in action and will read both namespaces. I'll do the changes

Reply:
ready for review